resources/executables/linux/ballerina

```diff
@@ -30,6 +30,11 @@
 # -----------------------------------------------------------------------------
 
 [ -f /etc/profile.d/wso2.sh ] && source /etc/profile.d/wso2.sh
+
+if [[ ( -z "${BALLERINA_HOME}" || ! -d "${BALLERINA_HOME}" ) && -d "/usr/lib64/ballerina" ]]; then
+    export BALLERINA_HOME="/usr/lib64/ballerina/"$(ls -1 /usr/lib64/ballerina/ | grep 'ballerina[0-9\.a-zA-Z]*' | head -1)
```

Comment:
Currently installers keep add new files for each release upon installation of each package. Would this `$(ls -1 /usr/lib64/ballerina/ | grep 'ballerina[0-9\.a-zA-Z]*' | head -1)` conflicts when there are several files for future releases? (eg:- ballerina-0.990.0, ballerina-0.991.0,  ballerina-0.991.0)

Also keeping distribution in lib64 directory on RHEL has been done via rpm and for ubuntu we keep distribution in lib directory via deb. 

As these locations are specific for OS and package manager, shall we leave these changes in installers and not include in this ballerina shell file as its a generic? WDYT?

Reply:
The path capture wouldn't be a problem for multiple distribution, but now that you mentioned I have updated it to pick latest distribution placed there, earlier it was lexical first pick.

I'm not sure where package manager repo code is, else I would have updated the resolution of BALLERINA_HOME there itself.